tests/codestyle/CodestyleTest.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+declare(strict_types=1);
+
+use Composer\Console\Application;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Console\Input\ArrayInput;
+
+class CodestyleTest extends TestCase
+{
+    protected const FIXTURES = [
+        "singleQuotes",
+        "strictTypes",
+        "unionTypes",
+    ];
+
+    protected Application $application;
+
+    protected function setUp(): void
+    {
+        $this->bootstrapComposer();
+    }
+
+    /**
+     * @throws Exception
+     */
+    public function testFixtures(): void
+    {
+        $this->clearTempDirectory();
```

Comment:
I think we could safely move this operation to `tearDown` function

```php
    protected function tearDown(): void {
        $this->clearTempDirectory();
    }
```

Reply:
Moved, but I moved it to `setUp()` too to be sure that broken tests won't fuck up anything.